### ISSUE TITLE
NullPointerException in ImageFragment

### ISSUE THREAD

**Issue by mbaleczny** [VEINHORN/ScrollGalleryView#35]
Sometimes when I swipe images I got NPE indicating this line in ImageFragment.class

```
...
@Override
    public void onSaveInstanceState(@NonNull Bundle outState) {
       ....
        outState.putParcelable(Constants.IMAGE, ((BitmapDrawable) backgroundImage.getDrawable()).getBitmap());
        ...
    }
```

Quick fix for that issue is adding IF

```
@Override
    public void onSaveInstanceState(@NonNull Bundle outState) {
        ...
        if (backgroundImage != null && backgroundImage.getDrawable() != null) {
            outState.putParcelable(Constants.IMAGE, ((BitmapDrawable) backgroundImage.getDrawable()).getBitmap());
        }
       ...
    }
```

Have anybody else encountered this error?

**Comment by madhacked** [comment]
Repro'd for me - basically if the image can't load for some reason, this will happen. Push a PR with the null check! Otherwise, you'd have to check the validity of the images before passing them to the gallery.

**Comment by mbaleczny** [comment]
Resolved in #42
